src/components/MovieBox.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, styled, Typography } from '@mui/material';
+
+const StyledBox = styled(Box)({
+    maxWidth: 300,
+    margin: 20,
+    padding: 10,
+    border: '3px solid #1e1e1e',
+    backgroundColor: '#e1e1e1',
+    borderRadius: 10,
+    '&:hover': {
+        border: '3px solid #00ffff'
+    }
+});
+
+const MovieBox = ({id, title, name, poster, overview}: {id: number, title: string, name: string, poster: string, overview: string}) => {
```

Comment:
This could be simpler:

```suggestion
const MovieBox = ({id: number, title: string, name: string, poster: string, overview: string}) => {
```

Reply:
I tried this locally, but for some reason it did not work...